releasenotes/notes/33472.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+  - 33472
+releaseNotes:
+  - |
+    **Added** `VERTIFY_CERT_AT_CLIENT` environmental variable to istiod. Setting `VERTIFY_CERT_AT_CLIENT` to `true` will verify server certificates using the OS CA certificates when not using a `DestinationRule` `caCertificates` field [Issue #33472](github.com/istio/istio/issues/33472).
```

Comment:
can they also set `caCertificates: system` explicitly?

If so (which I think they should?) we should add a warning to the webhook when we detect an explicit path to use `system` instead. Definitely should be a followup though, not in this PR. Lets get this in first

Reply:
They would be able to use `system` and it will treat it the same.